torch/csrc/distributed/c10d/init.cpp

```diff
@@ -1333,6 +1333,10 @@ Example::
           py::arg("master_listen_fd") = py::none(),
           py::arg("use_libuv") = false,
           py::call_guard<py::gil_scoped_release>())
+      .def(
+          "collect_client_counters",
+          &::c10d::TCPStore::collectClientCounters,
+          "Return a dict of counters for tcp store client")
```

Comment:
so the usage of  `collectClientCounters` in py is like:
```
## user code here ##
perf_data = tcp_store.collectClientCounters()
for fn_name in ["get", "set", "wait" , ...]:
    data = perf_data[fn_name]
    print(f"{fn_name}: count=data['count'], mean=data['mean'], sample_variance=data['sample_variance']")
```

Reply:
Almost, one should check if `sample_variance` is present since it might not be valid.
It makes me realize this is this a bad idea and I'll change the code to return NaN in those cases.